### PR TITLE
Message queue integrated.

### PR DESCRIPTION
**Pull request by manden-patinala**
I've integrated a message queue to support communication between operation and printer process.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
     "VARISCITE": {
 		/* Target Device Settings */
-		"TARGET_IP":"192.168.1.72",
```
Comment (erhbug) [inline .vscode/settings.json:4]: keep this ip

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+		"TARGET_IP":"192.168.1.82",
 
 		/* Project Settings */
-		"PROGRAM":"main",
+		"PROGRAM":"printer",
```
Comment (erhbug) [inline .vscode/settings.json:7]: ok

File: .vscode/settings.json (hunk 1 above)
Comment (manden-patinala) [reply .vscode/settings.json:4]: Sure, I'm going to change that line of code.

**Comment by erhbug** [comment]
Hi @manden-patinala, thank's for add the message queue, this is very important with us project!!!.

**Review by erhbug — APPROVED** [review]

**Pull request merged**